site/static/examples/kind-with-registry.sh

```diff
@@ -6,7 +6,7 @@ reg_name='kind-registry'
 reg_port='5001'
 if [ "$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true)" != 'true' ]; then
   docker run \
-    -d --restart=always -p "127.0.0.1:${reg_port}:5000" --name "${reg_name}" \
+    -d --restart=always -p "127.0.0.1:${reg_port}:5000" --net=kind --name "${reg_name}" \
```

Comment:
this doesn't work reliably because the network doesn't exist yet if you've never created any clusters, which is why we use "network connect"

Reply:
We could, perhaps run this container in `bridge` mode too? Like the `kind-control-plane` container?

I think the podman team is not allowing to "mix" network modes, see https://github.com/containers/podman/issues/13683#issuecomment-1081446705

Reply:
How would we do that? Creating another throwaway network for the registry?

Reply:
Specifying `--network bridge` is probably OK, I think docker always has this network but it's not clear if podman does or we'd have to create it.

Reply:
I think the `bridge` network will be there. It's well documented 
https://github.com/containers/podman/blob/main/docs/tutorials/basic_networking.md#bridge

Reply:
I updated the PR to this